boards/nrf51dk/src/main.rs

```diff
@@ -113,6 +113,7 @@ pub struct Platform {
     temp: &'static capsules::temp_nrf51dk::Temperature<'static, nrf51::temperature::Temperature>,
     rng: &'static capsules::rng::SimpleRng<'static, nrf51::trng::Trng<'static>>,
     aes: &'static capsules::symmetric_encryption::Crypto<'static, nrf51::aes::AesECB>,
+    radio: &'static capsules::ble::BLE<'static, nrf51::radio::Radio, VirtualMuxAlarm<'static, Rtc>>,
```

Comment:
It would be nice to name this something more descriptive than `radio`, maybe `ble_radio`